packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -285,6 +285,7 @@ export const graphFieldOptions = {
   stacking: [
     { label: 'Off', value: StackingMode.None },
     { label: 'Normal', value: StackingMode.Normal },
+    { label: 'Percent', value: StackingMode.Percent },
```

Comment:
```suggestion
    { label: '100%', value: StackingMode.Percent },
```
to match what google calls it?

Reply:
changed